test/middleware/spec.back-link.js

```diff
@@ -1,11 +1,20 @@
-var backLinks = require('../../lib/middleware/back-links');
-
-var util = require('util'),
+var proxyquire = require('proxyquire'),
+    util = require('util'),
     EventEmitter = require('events').EventEmitter;
 
+var getBackLinks = function (previousSteps) {
+    return proxyquire('../../lib/middleware/back-links', {
+        '../util/helpers': {
+            getPreviousSteps: function () {
+                return previousSteps;
+            }
+        }
+    });
+};
```

Comment:
It would also give you the rest of the sinon stub api so you could assert on other properties if need be.